src/main/java/jalau/at18/azul/CenterTileBoard.java

```diff
@@ -0,0 +1,30 @@
+package jalau.at18.azul;
+import java.util.*;
+
+public class CenterTileBoard extends ArrayList<Character> {
+
+
+    public CenterTileBoard() {
+        super();
+        this.add('F');
```

Comment:
Why do we need to add this tile in the constructor?

Reply:
Yes, you are right. It isn't really necessary. I will fix that